AdminApi.Sdk/Endpoints/Metrics/MetricsEndpoint.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Backbone.AdminApi.Sdk.Endpoints.Common;
+using Backbone.AdminApi.Sdk.Endpoints.Common.Types;
+using Backbone.AdminApi.Sdk.Endpoints.Identities.Types;
+
+namespace Backbone.AdminApi.Sdk.Endpoints.Metrics;
+
+public class MetricsEndpoint(EndpointClient client) : Endpoint(client)
+{
+    public async Task<AdminApiResponse<List<Metric>>> GetAllMetrics() => await _client.Get<List<Metric>>("Metrics");
```

Comment:
There should be a specific type for the response (`GetAllMetricsResponse`). Same for all other GetAll/List methods (just like you did it for the Consumer API response)